src/app/components/context-menu/context-menu.directive.ts

```diff
@@ -100,7 +100,7 @@ export class ContextActionsDirective implements OnInit, OnDestroy {
   }
 
   private getTarget(event: MouseEvent): Element {
-    return this.findAncestor(<Element>event.target, 'adf-datatable-cell');
+    return this.findAncestor(<Element>event.target, 'adf-datatable-cell--');
```

Comment:
why two dashes at the end? is that a class name?

Reply:
Because of the issue with nested 'adf-datatable-cell' elements coming from ADF, we check instead for any element that has a class name that contains the string 'adf-datatable-cell--' Such classes are not found on nested items to cause issues with opening the context menu.
I've opened a bug on ADF side to remove the  'adf-datatable-cell' on nested items of another 'adf-datatable-cell' item

Reply:
Ok, I'm putting this PR on hold until we have a clean solution coming from ADF.